src/content/_index.en.md

```diff
@@ -11,14 +14,39 @@ As Kubernetes gains adoption, teams are finding they must deploy and manage mult
 
 Submariner is completely open source, and designed to be network plugin (CNI) agnostic.
 
-
-## What Submariner Provides?
+### What Submariner Provides?
 
 * Cross-cluster L3 connectivity using encrypted VPN tunnels
 * Service Discovery across clusters via [Lighthouse](./architecture/components/lighthouse/)
 * [subctl](./deployment/), a friendly deployment tool 
 * Support for interconnecting clusters with [overlapping CIDRs](./architecture/globalnet/)
 
+<style>
+.mygrid {  
+    display: grid;
+    grid-gap: 12px;  
+    grid-template-columns: repeat(auto-fit, minmax(180px, 1fr));
+    grid-template-rows: repeat(2, 100px);  
+}
+
+</style>
+
+<div class="mygrid">  
+  <div>
+    <h5> Connecting containers with Submariner</h5>
+    <div> {{< youtube fMhZRNn0fxQ >}}</div>
+  </div>
+  <div>
+    <h5> Deploying Submariner
+</h5>
+    <div> {{< youtube cInmBXuZsU8 >}}</div>
+  </div>
+  <div>
+    <h5> Service discovery in Submariner</h5>
+    <div> {{< youtube tXsemQPNhyQ >}}</div>
+  </div>
+</div>
+
```

Comment:
Ahh. I see this is being omited, please add this to config.toml:

--- a/src/config.toml
+++ b/src/config.toml
@@ -6,6 +6,8 @@ title = "cross cluster network connectivity for Kubernetes"
 theme = "hugo-theme-learn"
 metaDataFormat = "yaml"
 defaultContentLanguageInSubdir= true
+[markup.goldmark.renderer]
+  unsafe = true
 
 [params]
   editURL = "https://github.com/submariner-io/website/edit/master/src/content/"

Reply:
added, hopefully I did it correctly.